cfn-resources/private-endpoint/cmd/resource/resource.go

```diff
@@ -336,3 +340,24 @@ func addModelToProgressEvent(progressEvent *handler.ProgressEvent, model *Model)
 
 	return *progressEvent
 }
+
+func CompareSlices(a []string, b []string) bool {
```

Comment:
Should be OK for this but the time complexity would be O(N^2). We can do better by converting arrays to map O(N).